### DISCUSSION TITLE
Abnormal selection box & objects shifting

### DISCUSSION THREAD

**Discussion by MeerukYang** [ppy/osu#31117] [Q&A]
I just select it and trying to change its position and it just messed up. = =

video linked:
[【osu!】lazer编辑器这是肿么肥四???](https://www.bilibili.com/video/BV1TfBVYsEBC/?vd_source=2e93116f26a2cb3077c48773ab5bafda&spm_id_from=333.788.videopod.episodes) P1

it seems that the circle is out of ratio, but it didn't show by ai mod. and notice there's a breaktime after the circle and the "time" in inspector was increasing way too high while i drag it.
however it doesn't allow out-of-ratio objects in lazer editor, so i assume it be a bug.

logs:
[compressed-logs.zip](https://github.com/user-attachments/files/18134955/compressed-logs.zip)

i save the beatmap project to export the logs and re-edited it and it just go back to normal; i'll never reproduce this i think. T^T

**Comment by bdach** [comment]
I can't see closely because you can't see 1080p on bilibili without an account. It looks like *maybe* grid spacing turned way up but not sure.

**Comment by MeerukYang** [comment]
This happened again just now.

video linked:
[【osu!】lazer编辑器这是肿么肥四???](https://www.bilibili.com/video/BV1TfBVYsEBC/?vd_source=2e93116f26a2cb3077c48773ab5bafda&spm_id_from=333.788.videopod.episodes&p=2) P2

For object selection, the selection area are larger than usual, as if there be an another object hiding behind the "edit" button is also selected; but actually only 1 object is selected.

As for object shifting, this could not happened to objects before some time point. for example in the linked video, P2, the "3" circle at 73375.3ms and objects before it would not shift; but objects after would.

**Reply by bdach** [reply]
I just noticed that the yellow selection boxes around the objects also look completely broken which I've never seen happen before. Can you like link the map or something so I can check it out?

**Reply by MeerukYang** [reply]
[The map used](https://osu.ppy.sh/beatmapsets/2289480#osu/4898972).(not updated since yesterday)
however this bug didn't happen after last time i saved and re-edited the map, i assume its not easily reproducible.

**Reply by bdach** [reply]
Yeah no idea, can't make it happen. Without a solid reproduction scenario this is going to be difficult to track down.
